### ISSUE TITLE
Task API executed without sandbox and without separated process

### ISSUE THREAD

**Issue by korshenko** [SubKit/subkit-microservice#69]
Task body could freeze microservice. 
```
while(1) {}
```
I think that each task must be executed in some kind of a sandbox in separated process (NodeJs fork, spawn and etc.).
Running each task in same process with microservice is dangerous.
Sandbox is extremely recommended for each task.

**Comment by MikeBild** [comment]
Yes, of course. Next NodeJs version supports timeout for VM run. Child process and messaging between processes is a second option. At this moment I prefer "don't create endless loops" - Of cause.

**Comment by MikeBild** [comment]
Is that issue solved?

**Comment by PerhunAndrey** [comment]
I believe that issue must be leave until the sandbox will be implemented.
